setup.py

```diff
@@ -42,7 +42,7 @@
 ]
 INSTALL_REQUIRES = [
     "biopython>=1.74,<2",
-    "numpy>=1.16,<2",
+    "numpy>=1.16.6,<2",
```

Comment:
I guess there is no need to set the requirement to this specific version? It seems anything < 1.17 should do.

Reply:
The idea was that we build with 1.16.6 (see GitHub Actions workflow), which retains forwards compatibility up to the current version. So users should be able to use any version numpy version from 1.16.6 and newer. Would you relax this to `numpy>=1.16,<3`?

Reply:
Yes, as we already fix the numpy version to build the wheels. Unless a wheel built using 1.16.6 doesn't work with e.g. 1.16.0, I would stick to >= 1.16. It shouldn't really matter, it's just seems a bit cleaner not to require a specific patch release unless it's actually necessary.